reset-aspel-worspace.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Step 1: Reset and clean all submodules
+for dir in packages/*; do
+  if [ -d "$dir/.git" ]; then
+    echo "-----------------------------------------------"
+    echo " 🚧  Resetting and cleaning $dir... Please wait!"
+    echo "-----------------------------------------------"
+
+    # Navigate to the submodule directory
+    cd "$dir"
+
+    # Revert any changes (reset to HEAD)
+    git reset --hard
+
+    # Remove untracked files (including node_modules and package-lock.json)
+    git clean -fdx
+
+    # Go back to the root directory
+    cd - > /dev/null
+  fi
+done
```

Comment:
we could probably prompt the developer to choose what they want to do. 

a. Select "A" to reset the workspace to the factory setting. 
b. Select "B" to ..... 

Ideas welcome!

Reply:
Yarn workspace prefers yarn-lock. When there is a package-lock file it can have an adverse effect on the installation of dependencies. 

There are Pros and Cons. 

I am inclined to the fact that it will benefit me clean sweeping the workspace. On the otherhand it has a drawback that some changes which we may want will also be wiped.